react/react1/week2/react1-week2-app/src/components/TodoTask.js

```diff
@@ -0,0 +1,30 @@
+function TodoTask(props) {
+  // id in li.id is exposed to later target it with remover. span.id is exposed for visual clarity for testing purposes.
+  return (
+    <li className="todoItemCard" id={props.id}>
+      <div>
+        <p className="taskText"> Task : {props.description} </p>
+        <span> ID = {props.id}</span>
+      </div>
+      <div className="cardControlls">
+        <label class="switch">
+          <input type={"checkbox"} onChangeCapture={handleChange}></input>
+          <span class="slider round"></span>
+        </label>
+        <button id="deleteTask" onClick={props.removeTodo}>
+          Delete
+        </button>
+      </div>
+    </li>
+  );
+}
+function handleChange(event) {
+  if (event.target.checked === true) {
+    console.log(`Done Handled`);
+    event.target.closest("li").classList.add("taskDone");
+  } else {
+    event.target.closest("li").classList.remove("taskDone");
+    console.log(`Undone Handled`);
```

Comment:
Something else to think about, is you could add a new attribute called 'status' or 'checked' to your todo object. 
Then, you could update it here in handleChange, and then depending on that set the classname you want:
```
<li className={todo.status ? "todoItemCard checked-item" : "todoItemCard"} id={props.id}>
```